src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -250,14 +115,14 @@ std::optional<int32_t> GlobalAppSettings::GetInitialY() const noexcept
 void GlobalAppSettings::ApplyToSettings(TerminalSettings& settings) const noexcept
 {
     settings.KeyBindings(GetKeybindings());
-    settings.InitialRows(_initialRows);
-    settings.InitialCols(_initialCols);
-    settings.RowsToScroll(_rowsToScroll);
-
-    settings.WordDelimiters(_wordDelimiters);
-    settings.CopyOnSelect(_copyOnSelect);
-    settings.ForceFullRepaintRendering(_forceFullRepaintRendering);
-    settings.SoftwareRendering(_softwareRendering);
+    settings.InitialRows(_InitialRows);

```

Comment:
Something something "underscore with capital letter is supposed to be reserved."

Reply:
Meh that's what our macros do and it's a giant annoying but simply mechanical change to fix it.

Reply:
I don't care that much. I just thought we had some slow semblance of trying to not introduce more of them. But if it's a pain in the ass, just forget it. It's not worth it and no one abides by it in practice.